pkg/tekton/pipelines.go

```diff
@@ -493,6 +496,33 @@ func ApplyPipeline(jxClient versioned.Interface, tektonClient tektonclient.Inter
 	return nil
 }
 
+// StructureForPipelineRun finds the PipelineStructure for the given PipelineRun, trying its name first and then its
+// Pipeline name, returning an error if no PipelineStructure can be found.
+func StructureForPipelineRun(jxClient versioned.Interface, ns string, run *pipelineapi.PipelineRun) (*v1.PipelineStructure, error) {
+	structure, err := jxClient.JenkinsV1().PipelineStructures(ns).Get(run.Name, metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			// Try again but with the Pipeline name for this run.
+			pipelineName := run.Labels[pipeline.GroupName+pipeline.PipelineLabelKey]
```

Comment:
Why do you combine `pipeline.GroupName+pipeline.PipelineLabelKey` to search for the label?

Reply:
To get the combination of `tekton.dev` and `/pipeline` for the full label. =)